checker/left_quantifiers.go

```diff
@@ -0,0 +1,55 @@
+package checker
+
+import (
+	"fmt"
+	"go/ast"
+)
+
+func init() {
+	must(Register("left_quantifiers", NewLeftQuantifiersChecker))
```

Comment:
You can add the left_quantifiers rule to the lingo config used for this project :)

Reply:
Done.